### PR TITLE
Made calculation of bond kinetic energies simpler, replaced calculation of bond rotational velocity with bond omega, and made error message if not comm_modify vel yes

### PR DESCRIPTION
**Pull request by efremdan1**

### CODE REVIEW

**Comment by akohlmey** [comment]
FYI, this code fails to compile. See, e.g., https://ci.lammps.org/job/lammps/job/pull-requests/job/openmpi-pr/68/console

**Comment by efremdan1** [comment]
Working on the compilation issue.

**Comment by efremdan1** [comment]
Fixed the compilation error. I had accidentally changed strcomp to stxcomp.

File: src/compute_bond_local.cpp [hunk 1]
```diff
@@ -86,6 +88,8 @@ void ComputeBondLocal::init()
 {
   if (force->bond == NULL)
     error->all(FLERR,"No bond style is defined for compute bond/local");
+  if (comm->ghost_velocity == 0)
```
Comment (akohlmey) [inline src/compute_bond_local.cpp]: @efremdan1 This test should only be made, if the user requested a property requiring ghost velocities. For just potential energy or force, ghost velocities are not needed.

Comment (efremdan1) [inline src/compute_bond_local.cpp]: Yep. Done.

**Comment by akohlmey** [comment]
still one problem: you have to move the test for ghost atom velocities back to the ::init() method.
It doesn't matter, if ghost atom velocities are enabled when the compute command is issued (and the class instantiated). what matters are to check right before the compute is actually being used. it is totally valid to first define the compute, and then enable ghost atom velocities. same is true for defining the compute before having defined a bond style. similarly, you have to protect against scripts where required things are disabled between instantiating the compute and using it.

**Pull request merged**